lib/rules/media-feature-no-invalid/README.md

```diff
@@ -0,0 +1,57 @@
+# media-feature-no-invalid
+
+Disallow invalid media queries.
+
+<!-- prettier-ignore -->
+```css
+@media not(min-width: 300px) {}
+/**    ↑
+ * This media query */
+```
+
+Media queries must be grammatically valid according to the [Media Queries Level 5](https://www.w3.org/TR/mediaqueries-5/) specification.
```

Comment:
I am unsure if this is the best way to explain this rule.  
I know this feature too well on a deeply technical level, so it isn't easy for me to explain it well for CSS authors.

Suggestions to improve would be very welcome :)

Reply:
If you think that this rule will be updated immediately after the release of the hypothetical level 6 specification then you can leave it as is.
If it's unlikely then something more vague would give us more leeway if that happened.